php/phalcon/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM php:7.4-fpm-alpine
```

Comment:
This file is not only for `phalcon`, but for all **php** frameworks (should not be versioned)

Reply:
how then can I inject configs into php.ini?

Reply:
Unfortunately, you can't right now. The only exception is that this configuration is the same for all frameworks (in this case you could add entry in `php/Dockerfile`). I'll create a `PR` if needed

Reply:
yeah i can create such common config